CesiumGeospatial/include/CesiumGeospatial/Projection.h

```diff
@@ -12,6 +12,8 @@
 
 namespace CesiumGeospatial {
 
+class Ellipsoid;
```

Comment:
This forward declaration isn't sufficient in the usual case that `CESIUM_DEFAULT_ELLIPSOID` is set to ` = CesiumGeospatial::Ellipsoid::WGS84`. So we can definitely just remove it, and we may want to add the actual header. We're probably getting away without the include because one of the other headers includes Ellipsoid.h.